packages/react-scripts/config/webpack.config.prod.js

```diff
@@ -97,6 +83,46 @@ module.exports = {
         .relative(paths.appSrc, info.absoluteResourcePath)
         .replace(/\\/g, '/'),
   },
+  optimization: {
+    minimizer: [
```

Comment:
I might make sense to add a CSS minimizer here too, instead of `minimize: true` in css-loader.

This would do a whole file minimization instead of a per-module minimization in the css-loader.

Reply:
Has this been resolved by adding the `OptimizeCSSAssetsPlugin` below?

Reply:
@iansu yes, we're now doing whole file minimization via optimizecssassetsplugin